README.md

```diff
@@ -115,11 +115,29 @@ Classes that are shared by the clients and servers.
 
 Some basic infrastructure classes used throughout the project.
 
-## OUR CHANGES (Team 27)
+## OUR CHANGES & NEW FEATURES (Team 27)
 
 - Swapped out the Java Swing client GUI with a JavaFX version
-- Updated the client-side and server-side controllers to interface with new GUI and database
-- Integrated a DERBY local database to save user's database
+- Created a sign-in screen seperate from the main chat
+- Requires a password and checked against No-SQL injection & invalid characters
+![alt text](https://raw.githubusercontent.com/GrayTurtle/codeu_project_2017/develop/images/SignIn.png)
+- Revamped the chat to be more user friendly
+- Chat updates automatically without the update button
+- Chat updates for other clients
+- The current user does not see themselves in the user list
+- Conversations reorder themselves based on most recent message sent
 - User's names are displayed in different colors depending on how many messages they've sent
+- 5 messages sent turns the user's name to red, 10 to blue, and 15 to gradient
+![alt text](https://raw.githubusercontent.com/GrayTurtle/codeu_project_2017/develop/images/Chat.png)
+- Updated the client-side and server-side controllers to interface with new GUI and database
+- Integrated a DERBY local database to save chat information
+- The chat is saved, and once users sign in it is the same as they had left it
+
 
 ## PLANNED CHANGES
+
+- Implement leaving a conversations (we currently just have a button for it)
+- Make the client update faster by changing out the timer for something else
```

Comment:
Like instead having it update every second or something, update whenever it needs to? I remember you talking about something like that